pkgs/tools/misc/bonfire/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, python3Packages, fetchFromGitHub }:
+
+with python3Packages;
+
+buildPythonApplication rec {
+  version = "0.0.7-20170119";
```

Comment:
In this case version should just be `20170119`, unless this versioning scheme is recommended somewhere?

Reply:
Thanks for reviewing @vyp . Rebased changes with just 20170119.

Reply:
Actually it should be `2017-01-19`: https://nixos.org/nixpkgs/manual/#sec-package-naming